process/versioning/README.md

```diff
@@ -6,17 +6,75 @@ This area also captures the git branching strategy to follow when developing pat
 
 ## Versioning guidelines
 
-TODO: What should our general versioning guidelines be? Do we distinguish between user-facing products versus development-focused libraries (and mostly interval vs external dev facing)? Do we want to push for semantic versioning?
+We use [semantic versioning](https://semver.org/). Initial releases of a major version will be proceeded by one or more pre-release versions. Pre-release versions, know as release candidates, use the identifier `rc[0-9]+`.
 
-### Versioning rules for the EOSIO product suite
+## Code Merge Standards and Processes
 
-TODO: Specific rules we have for the EOSIO product suite which includes nodeos, keosd, cleos, etc. as well as CDT and the system contracts (not clear if eosjs or other client libraries should be considered part of that core suite). Discuss hwo we want to maintain consistent major version across the products in the suite, but minor and patch versions remain independent.
+Main must be releasable. Features should be merged when they are ready to be released. New features should be tested and approved.
 
-## Branching strategy
+### Merge Details
+For a Pull Request (PR) identify the earliest Major/Minor release you are targeting.
 
-TODO: Discuss `main` branch and `release/*` branches. Discuss how patches to `release/*` branches must be promoted in a particular way to `main`, with examples for clarity. What about building and testing for the next major release so that `main` can remain head of development even as we continuously integrate features for the next major (possibly breaking release) while still being ready to cut a new minor release as needed?
+* Start the PR name with the release in brackets example “[3.2] ”
```

Comment:
We only need these for PRs associated with issues that are part of a Patch Milestone (see https://github.com/eosnetworkfoundation/engineering/blob/main/process/work_tracking/milestones.md). For Feature Milestone PRs, there is no need for the square bracket prefix.